docs/install.md

```diff
@@ -185,8 +185,6 @@ ntfy is packaged in nixpkgs as `ntfy-sh`. It can be installed by adding the pack
 nix-env -iA ntfy-sh
 ```
 
-NixOS also supports [declarative setup of the ntfy server](https://search.nixos.org/options?channel=unstable&show=services.ntfy-sh.enable&from=0&size=50&sort=relevance&type=packages&query=ntfy). 
-
```

Comment:
Your branch seems to be out of date. Pull the latest changes via `git pull upstream main` ("upstream" is the remote https://github.com/binwiederhier/ntfy, Google `git remote add`).

Reply:
Line restored